examples/fluent-bit/dynatrace/task-definition.json

```diff
@@ -0,0 +1,45 @@
+{
+    "family": "firelens-example-dynatrace",
+    "taskRoleArn": "arn:aws:iam::XXXXXXXXXXXX:role/ecs_task_iam_role",
+    "executionRoleArn": "arn:aws:iam::XXXXXXXXXXXX:role/ecs_task_execution_role",
+    "containerDefinitions": [
+        {
+            "essential": true,
+            "image": "amazon/aws-for-fluent-bit:stable",
```

Comment:
Would you mind updating to our public ECR image: https://github.com/aws/aws-for-fluent-bit#amazon-ecr-public-gallery

```
public.ecr.aws/aws-observability/aws-for-fluent-bit:stable
```

For AWS customers, they can pull these images without any pull limits, whereas, this is not the case with Docker Hub.